scripts/gen_eval_files.lua

```diff
@@ -199,14 +199,6 @@ local function process_params(params)
   return params
 end
 
---- @class vim.gen_vim_doc_fun
---- @field signature string
---- @field doc string[]
---- @field parameters_doc table<string,string>
---- @field return string[]
---- @field seealso string[]
---- @field annotations string[]
```

Comment:
@lewis6991 do we need this class ? seems like we are using `vim.EvalFn` instead

Reply:
If it's the same then remove. Can't fully remember but I think I added this as there was a slight difference with one of the fields. I think the other class has more fields that aren't used here.